sametime/meson.build

```diff
@@ -1,14 +1,20 @@
 feature = get_option('sametime')
 
-summary({'Sametime': feature}, section : 'Protocols')
+meanwhile_dep = dependency('meanwhile', version: ['>= 1.0.0', '< 2.0.0'],
+                           required: false)
```

Comment:
Should be able to pass `required: feature` to use the default error if enabled-and-not-found.